haralyzer/http.py

```diff
@@ -290,7 +290,8 @@ def text(self) -> str:
         :return: Response body
         :rtype: str
         """
-        return self.raw_entry["content"]["text"]
+        content = self.raw_entry["content"]
+        return content.get("_textBase64", content.get("text"))

```

Comment:
To pass tests and maintain backwards compatibility, this could probably be
```python 
content = self.raw_entry["content"]
return content.get("_textBase64", content["text"])
```

Reply:
On Apple products, this property may not exists - Leading into a key error from the `formatted` property

Reply:
We should probably update `formatted` property to handle `KeyError`. Currently, Chrome throws a key error for the `text` attribute, which is why we have split tests [code](https://github.com/haralyzer/haralyzer/blob/master/tests/chrome/test_chrome_entry.py#L136-L137).